inlong-tubemq/tubemq-client-twins/tubemq-client-cpp/example/producer/test_producer.cc

```diff
@@ -0,0 +1,156 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#include <stdio.h>
+#include <string>
+#include <set>
+#include <thread>
+#include <chrono>
+#include <iostream>
+
+#include "tubemq/tubemq_client.h"
+#include "tubemq/tubemq_config.h"
+#include "tubemq/tubemq_errcode.h"
+#include "tubemq/tubemq_message.h"
+#include "tubemq/tubemq_return.h"
+#include "utils.h"
+
+using namespace std;
+using namespace tubemq;
+
+using tubemq::ProducerConfig;
+using tubemq::TubeMQProducer;
+
+struct MessageSentCallback {  
+  static AtomicLong TOTAL_COUNTER;
```

Comment:
we use google code style to codeing. Const name needs use example like kTotalCounter.